docs/guides/SimpleStartupGuide.md

```diff
@@ -0,0 +1,414 @@
+# USB Camera Device Service Simple Startup Guide
```

Comment:
There is no license listed.

Reply:
I don't believe the license is needed in these guides, only the readme... @ajcasagrande can you comment?